oio/event/filters/webhook.py

```diff
@@ -67,8 +67,29 @@ def process(self, env, cb):
                 'creationTime': alias['ctime'],
                 'modificationTime': alias['mtime'],
                 'version': alias['version'],
-                'metadata': {},
-            },
+                'metadata': properties,
+            })
+        else:
+            all_properties = self.container_client.content_get_properties(
+                account=url['account'], reference=url['user'],
+                content=url['content'])
```

Comment:
Isn't there a version?

Reply:
In the URL, there is no version.

Here is an example:
```
"url":{"ns":"OPENIO","account":"myaccount","user":"toto","path":"magic","content":"EEC419C19E76050065243D428D5FFA00","id":"8B78B3245B74710F3ACC1BEF4978E621F5E764E01FFB5621D23C4EECA2B7BB3D"}
```

Also, I already have the content ID. So the version is useless here.